package.build.ts

```diff
@@ -569,6 +569,12 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
                       { items: Record<string, { text: string; command: string; args?: any[] }>}>,
         },
 
+        "dance.useSystemClipboard": {
+          type: "boolean",
+          default: true,
+          description: "Allow default yank register (\") to map to system clipboard",
```

Comment:
```suggestion
          description: "Controls whether the default yank register (`\"`) maps to the system clipboard.",
```

To match how VS Code setting descriptions are phrased.